### PR TITLE
Tag FlatBuffers.jl v0.5.0

### PR DESCRIPTION
**Pull request by attobot**
Repository: [JuliaData/FlatBuffers.jl](https://github.com/JuliaData/FlatBuffers.jl)
Release: [v0.5.0](https://github.com/JuliaData/FlatBuffers.jl/releases/tag/v0.5.0)
Travis: [Travis Build Status](https://travis-ci.org/JuliaData/FlatBuffers.jl/builds/466740682)
Diff: [vs v0.4.0](https://github.com/JuliaData/FlatBuffers.jl/compare/10a758b4a9f15f750d13e818b6099036d32d722d...7159781dc1605dbfdec13e62dd2a258ede6a62ff)
`requires` vs v0.4.0: 
```diff
--- 0.4.0/requires
+++ 0.5.0/requires
@@ -1 +1 @@
-julia 0.7-
+julia 0.7 1.1-
```
cc: @rjkat

Please make sure that:
- CI passes for supported Julia versions (if applicable).
- Version bounds reflect minimum requirements.

### CODE REVIEW

**Comment by quinnj** [comment]
@rjkat, the results of JuliaCIBot are showing that both FeatherLib & Feather tests are now failing with the new FlatBuffers release. Were there breaking changes made in this release? Would you mind taking a look to see why they fail now? It's usually best practice to at least notify downstream authors of breaking changes so they can cap release versions on their packages and make updates as necessary.

**Comment by rjkat** [comment]
Sure thing. Breaking changes are unexpected. I will investigate.

**Comment by rjkat** [comment]
Ah, looks like it's a `REQUIRE` vs `Project.toml` thing. I added the `Parameters` dependency to `Project.toml` but not `REQUIRE`...

**Comment by rjkat** [comment]
Please close this pull request, I'll tag another 0.5.1 release.

**Comment by andreasnoack** [comment]
You can just remove that tag and reapply it once you are ready. That will automatically update this PR.

**Comment by rjkat** [comment]
Apologies, looks like there were some other test failures. Looks like the FlatBuffers package is missing some test cases that are used by Feather (namely empty objects & undefined fields). Third time lucky?
